docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md

```diff
@@ -127,6 +127,12 @@ Added a new object that represents the properties of an appointment or message i
 
 **Available in**: Outlook on Windows (connected to Office 365)
 
+#### [Office.context.mailbox.item.getItemIdAsync](office.context.mailbox.item.md#getitemidasyncoptions-callback)
```

Comment:
You added this method within the **Delegate access** section -- is that right? (Seems to me that this method isn't necessarily related to Delegate access.)

Reply:
It is being implemented to support a scenario where the delegate has only read access. So while it's not necessarily related to delegate access, it still seemed to make sense to include here.